app/build.gradle

```diff
@@ -1,6 +1,7 @@
 buildscript {
     repositories {
         maven { url 'https://plugins.gradle.org/m2/' }
+        maven { url 'https://jitpack.io' }
```

Comment:
I think this was needed at one point but it doesn't seem to be needed anymore.

Reply:
Done